src/Xamarin.Android.Build.Tasks/Tasks/LinkAssemblies.cs

```diff
@@ -146,13 +146,6 @@ bool Execute (DirectoryAssemblyResolver res)
 
 				var copydst = OptionalDestinationDirectory ?? OutputDirectory;
 
-				foreach (var pdb in PortablePdbFiles) {
```

Comment:
Given that, afaict, `PortablePdbFiles` is no longer being used, should this also just remove the `PortablePdbFiles` member, and update `Xamarin.Android.*.targets` to no longer provide that property?